sssom/util.py

```diff
@@ -146,14 +146,18 @@ def __str__(self) -> str:  # noqa:D105
     def clean_prefix_map(self) -> None:
         """Remove unused prefixes from the internal prefix map based on the internal dataframe."""
         prefixes_in_map = get_prefixes_used_in_table(self.df)
```

Comment:
I know this is not part of the PR, but it is part of the confusion here:

1. rename `prefixes_in_map` to `prefixes_in_table`. This variable has nothing to do with the map!